config/schema_descriptions.yml

```diff
@@ -6,52 +6,52 @@ reporting_patient_states:
     status: "active, dead, migrated, etc"
     gender: "Gender of the patient"
     age: "Age of the patient as entered by a nurse"
-    age_updated_at: "Time (in UTC) at which the age was last updated"
+    age_updated_at: "Time (in UTC) at which the field 'age' was last updated"
     date_of_birth: "Date of birth of the patient"
-    current_age: "Computed age as of today, given age, age-updated-at and date-of-birth. This will have the same value for a given patient across all rows."
-    month_date: "Date at the beginning of the month for which this row has the latest information of a patient"
+    current_age: "Patient's age as of today, based on age, age-updated-at and date-of-birth. This will have the same value for a patient across all rows."
```

Comment:
Let's wrap these field references in single quotes as well for consistency. Shall we use snake case as well?